cunumeric/random/random.py

```diff
@@ -28,22 +28,15 @@
     from ..types import NdShapeLike
 
 
-def seed(init: Union[int, None] = None) -> None:
-    """
-    (Re-)seed the global random number generator.
+default_rng = generator.default_rng
 
-    Parameters
-    ----------
-    init : int, optional
-        The seed number to use
 
-    See Also
-    --------
-    numpy.random.seed
+def seed(init: Union[int, None] = None) -> None:
+    """
+    Reseed the legacy random number generator.
 
-    Availability
-    --------
-    Multiple GPUs, Multiple CPUs
+    This function is effective only when the cuRAND is NOT used in the build
```

Comment:
typo: remove "the"

Reply:
[79eafc6](https://github.com/nv-legate/cunumeric/pull/497/commits/79eafc6cfacc8f7e07a5a3fd93aa033eafbf9944)